frontend/src/Settings/Indexers/Indexers/IndexersConnector.js

```diff
@@ -2,14 +2,14 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { createSelector } from 'reselect';
-import sortByName from 'Utilities/Array/sortByName';
+import sortBySearchPriorityThenName from 'Utilities/Array/sortBySearchPriorityThenName';
 import createSortedSectionSelector from 'Store/Selectors/createSortedSectionSelector';
 import { fetchIndexers, deleteIndexer, cloneIndexer } from 'Store/Actions/settingsActions';
 import Indexers from './Indexers';
 
 function createMapStateToProps() {
   return createSelector(
-    createSortedSectionSelector('settings.indexers', sortByName),
```

Comment:
Not sure, I feel like there's an argument for both. I do somewhat prefer sort by name but that's quite probably bias since I don't have 20 indexers in the first place. Not exactly the hill I want to die on.
I do agree that it should be consistent with download clients.
So... verdict, yeah sort by priority then name would be ok. Assuming it resorts properly on changes.

Also 'sortByPriorityThenName' would be even better since there's nothing 'indexey' in the function. (can we get it down to 'sortByMagic'?)

PS: This function needs to change if we change to the +1 -1 scheme.